### ISSUE TITLE
False positives from Test-ForUnusedVariable

### ISSUE THREAD

**Issue by nzbart** [StartAutomating/ScriptCop#8]
It appears that `Test-ForUnusedVariable` is returning false positives:

```
Test-Command -ScriptBlock { function UseVariable { $someVariable = 'hi'; $someOtherVariable = 'bye'; Compare-Object $someVariable $someOtherVariable } } -Rule Test-ForUnusedVariable
```

What's the best solution to this problem? Is there a way to suppress individual warnings?
